src/components/groups/group.routes.ts

```diff
@@ -59,4 +59,14 @@ router.delete('/:id',
   (req, res) => res.status(204).send('Csoport sikeresen törölve')
 )
 
+router.get('/copy/:id', isAuthenticated, getGroup, (req, res) =>
```

Comment:
I'd change the route to `/:id/copy`, since the copy functionality applies to the specific group, so it should be under that in the URI hierarchy.